src/main/resources/templates/db_scripts/schema.json

```diff
@@ -550,14 +550,22 @@
       "foreignKeys": [
         {
           "fieldName": "poLineId",
-          "targetTable": "po_line"
+          "targetTable": "po_line",
+          "targetTableAlias": "poLine",
+          "tableAlias": "pieces"
         },
         {
           "fieldName": "titleId",
           "targetTable": "titles",
           "targetTableAlias": "titles",
           "tableAlias": "pieces",
           "tOps": "ADD"
+        },
+        {
+          "targetPath": ["poLineId", "purchaseOrderId"],
+          "targetTable": "purchase_order",
+          "targetTableAlias": "purchaseOrder",
+          "tableAlias": "pieces"
```

Comment:
to knowledge sharing, what is purpose of this target path

Reply:
> to knowledge sharing, what is purpose of this target path

These are join columns, by which the ids are compared